frontend/src/pages/404.js

```diff
@@ -1,7 +1,21 @@
-import { Typography } from "@mui/material";
+import { Button, Typography } from "@mui/material";
 
 const NotFound = () => {
-  return <Typography variant="h3">404 - Page Not Found</Typography>;
+  return (
+    <div>
+      <Typography variant="h3">Have You Seen this page?</Typography>
+      <img
```

Comment:
Can you use [next/image](https://nextjs.org/docs/api-reference/next/image) instead of the default img tag?

Reply:
And with this, could you also improve the image for mobile usage?